docs/CONTRIBUTING.md

```diff
@@ -35,6 +35,10 @@ We're always looking for contributors.
 
 To run Argo Workflows locally for development: [running locally](running-locally.md).
 
+### Committing
+
+To commit Argo Workflows for PR: [Committing](running-locally.md#committing).
```

Comment:
```suggestion
See the [Committing Guidelines](running-locally.md#committing).
```

Reply:
@agilgur5 

I added a commit adding a link to signoff.
And your suggestion is good :)
Thank you for your sincere review!